.github/workflows/sync-translated-content.yml

```
@@ -59,6 +59,36 @@ jobs:
         working-directory: ${{ github.workspace }}/mdn/content
         run: yarn content sync-translated-content ${{ matrix.lang }}
 
+      - name: Gather related upstream commits
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # get upstream head sha
+          UPSTREAM_HEAD_SHA=$(cd "${{ github.workspace }}/mdn/content" && git rev-parse HEAD)
+          echo "upstream head sha: ${UPSTREAM_HEAD_SHA}"
+          # get original path of moved files
+          MOVED_FILES=$(git diff --cached --name-status --diff-filter=R | cut -f2 | sed "0,/${{ matrix.lang }}/s//en-us/")
```

Could you explain why the pattern `0,/ja/s//en-us/`? In my local it changed only the first line of the output.

Can we have a simpler version:
```suggestion
          MOVED_FILES=$(git diff --cached --name-status --diff-filter=R | cut -f2 | sed "s|files/${{ matrix.lang }}|files/en-us|")
```

---

my neglect! fixed in d78ef29982b98b07cfcdb5ae56bab29f72c90fed